_docs/style/typography.md

```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
+{% endfigure %}
 {% endcolumn %}
 </div>
 
-### Subdued style
-
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-You can use the subdued style with any text 14px and taller, and is used to let more important text stand out. The subdued style is not the same as an inactive link or button.
+### Link
 
-*Do not subdue text on backgrounds darker than Gray 25 (#fafafa) or it will not pass accessibility standards.*
+Links support styles for hover, active, and disabled states, and inherit size and spacing styles from the text to which they are applied. Helix does not support the visited state.
 
 {% endcolumn %}
 
 {% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
-{% include type-styles.html htmlexample='<p class="hxSubdued">Subdued Style</p>' typeface="Roboto" csscode="Gray 900" hex="#424242" %}
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/style/typography/type-link.png){:width="635px"}
+{% endfigure %}
 {% endcolumn %}
-
 </div>
 
-### Sub-body text
-
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-Use sub-body text sparingly, since it is the least readable size. You can use it when smaller text is appropriate, such as for icon labels.
+### Caption
+
+Captions apply to images and visualizations. They should be used to clarify images and not introduce new, pertinent information.
 
 {% endcolumn %}
 
 {% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
-{% include type-styles.html htmlexample='<p class="hxSubBody">Sub-Body Text</p>' typeface="Roboto" weight="Regular (400)" fontsize="12px / 0.75rem" csscode="Gray 900" hex="#424242" linespace="1.5" %}
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/style/typography/type-caption.png){:width="635px"}
+{% endfigure %}
 {% endcolumn %}
 </div>
 
-### Caption
+<div class="hxRow" markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Sub-body text
+
+Use sub-body text sparingly, since it is the least readable size. For example use this typography style for the following:
+
+- Labeling iconography
+- Labeling the legend on visualizations
+- Providing tertiary details or cards and tiles
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/style/typography/type-sub-body.png){:width="635px"}
+{% endfigure %}
+{% endcolumn %}
+</div>
 
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-Captions apply to images and visualizations. They should be used to clarify images and not introduce new, pertinent information.
+### Subdued style
+
+Use the subdued style to allow more important information stand out. For example:
```

Comment:
s/`information stand`/`information to stand`

Reply:
updated.